<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogNewDownload</class>
 <widget class="QDialog" name="dialogNewDownload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a new download</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" alignment="Qt::AlignRight">
    <widget class="QDialogButtonBox" name="okButton">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="downloadDir">
     <property name="placeholderText">
      <string>Destination</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="url">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Link</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>accepted()</signal>
   <receiver>dialogNewDownload</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>rejected()</signal>
   <receiver>dialogNewDownload</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
